### ISSUE TITLE
Switch Theme Doesn't Work At First Click

### ISSUE THREAD

**Issue by realrufans** [kognise/water.css#65]
## Expected behavior
I was expecting it to switch the theme after clicking on the button the very first time.
## Actual behavior
The switch button doesn't work at first click. But works fine afterward.

## How to reproduce
- Go to https://watercss.netlify.com/
- click on Switch theme
- Watch it not switch at first click.

## Browser
Version 73.0.3683.103 (Official Build) (64-bit)

Operating system: Tecno Camon X pro ( Android 0)

# Recording Of The Bug

![im4.ezgif.com](https://im4.ezgif.com/tmp/ezgif-4-01bc94eb51f7.gif)

**Comment by pawptart** [comment]
Also not working on Chrome 74.0.37299.112 / Android 8.0.0

**Comment by realrufans** [comment]
https://youtu.be/PJBZR3u7dmM

**Comment by pawptart** [comment]
Did some testing, also was not working on Chrome on desktop. Looks like the bug is in `index.html` on Line 5.

https://github.com/kognise/water.css/blob/770040da41a8cb32205dec48c2a3678041892b50/index.html#L5

Taking a look at `script.js`:

https://github.com/kognise/water.css/blob/770040da41a8cb32205dec48c2a3678041892b50/script.js#L3-L6

Looks like it's expecting the `href` attribute on the stylesheet to be `dist/dark.min.css` instead of `dist/dark.css`. Should be a simple fix. Will submit a PR later.

**Comment by jonaskuske** [comment]
An alternative would be to do:


```js
stylesheet.href = stylesheet.href.replace(/dark|light/, replaced => replaced === 'dark' ? 'light' : 'dark')
```

This way only the dark/light part is replaced and other filename changes won't affect the switch function :)

**Comment by pawptart** [comment]
I don't see any reason not to implement your fix. It would be 1 line of code versus 5 currently.

**Comment by jonaskuske** [comment]
Well thinking about it, it'd need to be

```js
stylesheet.href = stylesheet.href.replace(/dark|light/, function(replaced) { return replaced === 'dark' ? 'light' : 'dark' })
```

...because judging by #61 the demo site needs to support Internet Explorer, and IE doesn't support arrow functions.

**Comment by kylejrp** [comment]
Closed by #66. Thanks for reporting @realrufans!
